src/model/account/Address.ts

```diff
@@ -25,12 +26,15 @@ export class Address {
      * Create from private key
      * @param publicKey - The account public key.
      * @param networkType - The NEM network type.
+     * @param {SignSchema} signSchema The Sign Schema. (KECCAK_REVERSED_KEY / SHA3)
      * @returns {Address}
      */
-    public static createFromPublicKey(publicKey: string, networkType: NetworkType): Address {
+    public static createFromPublicKey(publicKey: string,
+                                      networkType: NetworkType,
+                                      signSchema = SignSchema.SHA3): Address {
         const publicKeyUint8 = convert.hexToUint8(publicKey);
         const address = AddressLibrary
-            .addressToString(AddressLibrary.publicKeyToAddress(publicKeyUint8, networkType));
+            .addressToString(AddressLibrary.publicKeyToAddress(publicKeyUint8, networkType, signSchema));
```

Comment:
how does the signature schema affect public key to address algorithm ?

Reply:
hmm, good question. as we need an option to allow choosing either Keccak256 or sha3_256, but not necessarily need to use SignSchema, we can use the existing `HashType` instead. ??

Reply:
okay, agreed to use `SignSchema` here